spine-badge-css.scss

```diff
@@ -0,0 +1,23 @@
+@import '@material/elevation/mixins';
+
+
+:host {
+  display: inline-block;
+  --spine-badge-size: 21px;
+
+  min-width: var(--spine-badge-size);
+  height: var(--spine-badge-size);
+  border-radius: calc(var(--spine-badge-size) / 2);
+  padding: 0 6px;
+  box-sizing: border-box;
+  line-height: var(--spine-badge-size);
+  white-space: nowrap;
+
+  font-weight: 400;
+  font-size: 11px;
+  background: var(--primary-background-color, #ffffff);
```

Comment:
[`-primary-background-color`](https://github.com/PolymerElements/paper-styles/blob/master/default-theme.html#L26) (as well as the [`--primary-color`](https://github.com/PolymerElements/paper-styles/blob/master/default-theme.html#L35) below) belong to standard Polymer theme variables. Are there any analogs in MCW (or MCWC) that we can consider using? Let's discuss.